jemalloc-ctl/src/lib.rs

```diff
@@ -64,7 +64,7 @@
 //!     }
 //! }
 //! ```
-#![deny(missing_docs, broken_intra_doc_links)]
+#![deny(missing_docs, rustdoc::broken_intra_doc_links)]
```

Comment:
Just mention this will be rejected as a hard error by an old toolchain, though we never claimed the MSRV.

Reply:
Now they are suppressed instead. It will be updated to latest version when there is a minor version bump.